charts/consul/values.yaml

```diff
@@ -2195,6 +2195,17 @@ connectInject:
         # @type: string
         service: null
 
+      # The resource settings for Pods handling traffic for Gateway API.
```

Comment:
Jeff and I agreed that this made more sense to be set on the `managedGatewayClass`.